src/components/ColorPicker/ColorPickerConstants.ts

```diff
@@ -14,4 +14,6 @@ export const DIALOG_WIDTH_PADDING = 24;
 export const COLOR_SHAPES = {
```

Comment:
Why not convert color shapes to be regular enum?

Reply:
same as for the previous question regarding enum